x-pack/plugins/apm/server/lib/helpers/aggregated_transactions/get_fallback_to_transactions.ts

```diff
@@ -32,5 +36,52 @@ export async function getFallbackToTransactions({
     apmEventClient,
     kuery,
   });
-  return !searchesAggregatedTransactions;
+
+  if (!searchesAggregatedTransactions) {
+    // if no aggregrated transactions, check if any transactions at all
+    return await getHasTransactions({
+      start,
+      end,
+      apmEventClient,
+      kuery,
+    });
+  }
+
+  return false;
+}
+
+async function getHasTransactions({
+  start,
+  end,
+  apmEventClient,
+  kuery,
+}: {
+  start?: number;
+  end?: number;
+  apmEventClient: APMEventClient;
+  kuery: string;
+}) {
+  const response = await apmEventClient.search('get_has_transactions', {
+    apm: {
+      events: [ProcessorEvent.transaction, ProcessorEvent.span],
+    },
+    body: {
+      query: {
+        bool: {
+          filter: [
+            { exists: { field: TRANSACTION_DURATION } },
+            ...(start && end ? rangeQuery(start, end) : []),
+            ...kqlQuery(kuery),
+          ],
+        },
+      },
+    },
+    terminateAfter: 1,
+  });
+
+  if (response.hits.total.value > 0) {
+    return true;
+  }
+
+  return false;
```

Comment:
```suggestion
  return response.hits.total.value > 0;
```

if you want to spell it out:
```
  const hasTransactions =  response.hits.total.value > 0;
  return hasTransactions;